content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -234,22 +222,29 @@ model User {
 
-The `Restrict` action is **not** available on [Microsoft SQL Server](/concepts/database-connectors/sql-server) and will trigger a schema validation error. Instead, you can use [`NoAction`](#noaction), which produces the same result and is compatible with SQLServer.
+The `Restrict` action is **not** available on [Microsoft SQL Server](/concepts/database-connectors/sql-server) and triggers a schema validation error. Instead, you can use [`NoAction`](#noaction), which produces the same result and is compatible with SQL Server.
 
 </Admonition>
 
 ### <inlinecode>NoAction</inlinecode>
 
-The `NoAction` action is similar to `Restrict`, the difference between the two is dependant on the database being used:
+The `NoAction` action is similar to `Restrict`, the difference between the two is dependent on the database being used:
 
 - **PostgreSQL**: `NoAction` allows the check (if a referenced row on the table exists) to be deferred until later in the transaction. See [the PostgreSQL docs](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-FK) for more information.
 - **MySQL**: `NoAction` behaves exactly the same as `Restrict`. See [the MySQL docs](https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html#foreign-key-referential-actions) for more information.
 - **SQLite**: When a related primary key is modified or deleted, no action is taken. See [the SQLite docs](https://www.sqlite.org/foreignkeys.html#fk_actions) for more information.
-- **SQLServer**: When a referenced record is deleted or modified, an error is raised. See [the SQLServer docs](https://docs.microsoft.com/en-us/sql/relational-databases/tables/graph-edge-constraints?view=sql-server-ver15#on-delete-referential-actions-on-edge-constraints) for more information.
+- **SQL Server**: When a referenced record is deleted or modified, an error is raised. See [the SQL Server docs](https://docs.microsoft.com/en-us/sql/relational-databases/tables/graph-edge-constraints?view=sql-server-ver15#on-delete-referential-actions-on-edge-constraints) for more information.
+- **MongoDB** (in preview from version 3.6.0): When a record is modified or deleted, nothing is done to any related records.
+
+<Admonition type="warning">
+
+If you are [emulating referential integrity](/concepts/components/prisma-schema/relations/referential-integrity#handling-the-referential-integrity-in-prisma) in the Prisma client rather than using foreign keys in the database, you should be aware that currently Prisma only implements the referential actions. Foreign keys also create constraints, which make it impossible to manipulate data in a way that would violate these constraints: instead of executing the query, the database responds with an error. These constraints will not be created if you emulate referential intergrity in the client, so if you set the referential action to `NoAction` there will be no checks to prevent you from breaking the referential integrity.
```

Comment:
Need to double check that this information is also present on the referential integrity page.

Reply:
I'd moved it to the referential actions page as you suggested, but I've now added it back to referential integrity as well.